model-api-gen-gradle-test/typescript-generation/src/typeguards.test.ts

```diff
@@ -0,0 +1,49 @@
+import { useModelsFromJson } from "@modelix/vue-model-api";
```

Comment:
You do not need the `vue-model-api` for tests

You should be able to implement a helper `useModelFromJson` in this test only using `@modelix/model-client`.
Take a look at `vue-model-api/src/useModelsFromJson.ts`.

---

This could by the way solve the build issue you have.
I think I had similar issues when testing `model-api-gen-gradle-test/vue-integration`.